filament/src/details/MaterialInstance.h

```diff
@@ -53,7 +53,8 @@ class FTexture;
 
 class FMaterialInstance : public MaterialInstance {
 public:
-    FMaterialInstance(FEngine& engine, FMaterial const* material) noexcept;
+  FMaterialInstance(FEngine& engine, FMaterial const* material,
```

Comment:
nit: spacing here.

Reply:
Fixed spacing as suggested. Thanks for pointing that out!